src/Parser.js

```diff
@@ -48,7 +48,14 @@ export function parse (value, points, declarations, rules, rulesets) {
 				break
 			// /
 			case 47:
-				push(rulesets, comment(next()))
+				switch (peek(0)) {
```

Comment:
was thinking about moving this above main `default` case so we could fall through to it for this default case here, but then I couldn't use switch/case because I wouldn't be able to break 2x from this 42 and 47 case, so for now i've decided that it's not worth it

Reply:
What test does this fix?

Reply:
The one that is enabled in this PR - https://github.com/thysultan/stylis.js/pull/168/files#diff-910eb6f57886ca16c136101fb1699231R275

I've narrowed it down to:
```css
.user{w:2/1;}
```
being parsed to:
```css
.user{w:2;}
```
without this fix. Basically `/` has caused everything after it being treated as a comment.

Reply:
Let's use `font: 12px/14px sans-serif;` as the test for this.

Reply:
ok, added such. Other than that, is it OK to merge this in?